### PR TITLE
lyrebird: init at 1.1.0

### PR DESCRIPTION
**Pull request by OPNA2608**
First time I'm packaging something Python-y, please go easy on me if there are any problems with this. :stuck_out_tongue: 

###### Motivation for this change
New package "lyrebird" - an easy-to-use, GTK3-based voice changer application for Linux.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/lyrebird/default.nix [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{ python3Packages
+, lib
+, fetchFromGitHub
+, makeWrapper
```
Comment (dali99) [inline pkgs/applications/audio/lyrebird/default.nix:4]: This isn't needed I think

**Review by D4ndellion — CHANGES_REQUESTED** [review]

File: pkgs/applications/audio/lyrebird/default.nix (hunk 1 above)
Comment (OPNA2608) [reply pkgs/applications/audio/lyrebird/default.nix]: Fixed.

**Comment by wamserma** [comment]
Successfully built via `nix-shell -p nixpkgs-review --run "nixpkgs-review pr 96895"`.

**Comment by OPNA2608** [comment]
Those are normal, you'll need to put "Toggle Lyrebird" into the on position, then play around with the presets / pitch shift value while having something listening to the newly created "Lyrebird Virtual Input" device.

Tested with Discord, device gets picked up and successfully applies the selected presets.

**Comment by wamserma** [comment]
A warning for all possible reviewers: This PR was branched of while the [fontconfig font cache issue](https://github.com/NixOS/nixpkgs/issues/97418) was present in master. If the system behaves weird after `nix-review`, run `fc-cache -r`.

**Comment by OPNA2608** [comment]
Rebased on current master, I hope that helps with the fontconfig cache issue.

**Comment by Lassulus** [comment]
getting these errors when trying to use it: 
```
sox FAIL formats: can't open output file `Lyrebird-Output': can not open audio device: No such entity
No PulseAudio daemon running, or not running as session daemon.
Traceback (most recent call last):
  File "/nix/store/j475am9xsjzyvxjdgab6xmljj1l1dk2b-lyrebird-1.1.0/share/lyrebird/app/mainwindow.py", line 162, in toggle_activated
    null_sink = subprocess.check_call(
  File "/nix/store/bs03sg8b0gq2zr4v252hh9psp780qj5q-python3-3.8.5/lib/python3.8/subprocess.py", line 364, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['pacmd', 'load-module', 'module-null-sink', 'sink_name=Lyrebird-Output']' returned non-zero exit status 1.
```
wat do I need to do? otherwise looks good

**Comment by OPNA2608** [comment]
Added `pulseaudio` to PATH so it can find `pacmd`, blind guess that's what's missing? If that didn't fix it then I'd suspect the application having a problem with your audio setup: are you running Pulse & does `pacmd load-module module-null-sink sink_name=Lyrebird-Output` complete without errors if you run it?

**Comment by Lassulus** [comment]
hmm, it does indeed not work. I guess my setup is a little bit unconventional with pulseaudio running systemWide

**Comment by OPNA2608** [comment]
That does appear to be the problem, as lyrebird tries to load PulseAudio modules via `pacmd load-module`. Having a system-wide PulseAudio server seems to not be compatible with that approach, assuming none of the basic design behind this [*13 year old issue*](https://narkive.com/RyQTiLPL.6) has been changed.

> Normally the automatic module loading bit will fail with system PAs
> because module loading is not allowed and thus the module-cli-protocol-
> unix cannot be loaded.

> Either way, this is by design and I'm not sure it's something we should
> change, especially as system mode is not really recommended or really
> heavily tested upstream.

**Review by jonringer — APPROVED** [review]
diff LGTM
gui opens fine

```
https://github.com/NixOS/nixpkgs/pull/96895
1 package built:
lyrebird
```

**Pull request merged**